### ISSUE TITLE
Locals.sh: line 104: [: -ne: unary operator expected

### ISSUE THREAD

**Issue by lemmaa** [dotnet/coreclr#10344]
Unit test script give an error message `[: -ne: unary operator expected`.

```
Starting JIT/opt/perf/doublealign/Locals/Locals.sh
FAILED   - [   2][   1s]JIT/opt/perf/doublealign/Locals/Locals.sh
               BEGIN EXECUTION
               /home/sjlee/Windows_NT.x86.Release/Tests/coreoverlay.release/ildasm -raweh -out=Locals.dasm.il Locals.exe
               ./Locals.sh: line 104: [: -ne: unary operator expected
...
```
It's caused by typo in script, setting exit code to variable `$_ERRORLEVEL` and use `$ERRORLEVEL` in next condition statement.
```shell
# IlasmRoundTrip Script

echo "$CORE_ROOT/ildasm" -raweh -out=Locals.dasm.il Locals.exe
"$CORE_ROOT/ildasm" -raweh -out=Locals.dasm.il Locals.exe
_ERRORLEVEL=$?
if [  $ERRORLEVEL -ne 0 ]
then
  echo EXECUTION OF ILDASM - FAILED $ERRORLEVEL
  exit 1
fi

echo "$CORE_ROOT/ilasm" -output=Locals.asm.exe  Locals.dasm.il
"$CORE_ROOT/ilasm" -output=Locals.asm.exe  Locals.dasm.il
_ERRORLEVEL=$?
if [ $ERRORLEVEL -ne 0 ]
then
  echo EXECUTION OF ILASM - FAILED $ERRORLEVEL
  exit 1
fi  
```

**Comment by sdmaclea** [comment]
This was fixed by #10254

**Comment by lemmaa** [comment]
@sdmaclea Thanks!
